src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -98,12 +98,8 @@ public function apply(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $q
             if ($this->isPropertyNested($property)) {
```

Comment:
I've done this, removes a few lines in the `SearchFilter` but it still uses `addJoinOnce` on its own (l 153).